assets/ai2.yaml

```diff
@@ -115,3 +115,97 @@
   monitoring: Quality filtration, deduplication, and risk mitigation via logistic
     qualifiers and regular expressions used.
   feedback: ''
+
+- type: dataset
+  name: Tulu-V2-mix
+  organization: AI2
+  description: Tulu-V2-mix is a dataset composed of many high-quality instruction datasets that results in stronger performance across a variety of reasoning and knowledge-probing tasks.
+  created_date: 2023-11-20
+  url: https://arxiv.org/pdf/2311.10702.pdf
+  datasheet: https://huggingface.co/datasets/allenai/tulu-v2-sft-mixture
+  modality: text
+  size:
+    value: unknown
+    explanation: Magnitude of size is around 100M tokens, given the length distribution of dataset provided in model card.
+  sample: []
+  analysis: Models trained with dataset evaluated on downstream performance.
+  dependencies: [FLAN, Open Assistant 1, ShareGPT, GPT4 Alpaca, Code Alpaca, LIMA, WizardLM Evol Instruct, Open Orca]
```

Comment:
Can you update these names so they match the existing nodes we have, so the graph gets connected correctly? E.g. I am guessing GPT4 Alpaca should be Alpaca dataset?